__tests__/app.test.js

```diff
@@ -81,33 +81,85 @@ describe("GET articles by ID", () => {
 });
 
 describe("GET all articles", () => {
-  test("should respond with array of article objects", () => {
+  test("respond with array of article objects", () => {
     return request(app)
       .get("/api/articles/")
       .expect(200)
-      .then((response) => {  
-        const articles = response.body.articles
-        articles.forEach((article) => {
-          expect(article).toEqual({
-            article_id: expect.any(Number),
-            title: expect.any(String),
-            topic: expect.any(String),
-            author: expect.any(String),
-            created_at: expect.any(String),
-            votes: expect.any(Number),
-            article_img_url: expect.any(String),
-            comment_count: expect.any(String)
+      .then((response) => {
+        const articles = response.body.articles;
+        if (articles.length > 0) {
```

Comment:
this check should be a check for the exact length as the purpose of it is to check that there are some there and then check all the articles. this way if there were none then it would just skip the forEach and always pass